articles/step-by-step-on-how-to-user-nodejs-repl-tutorial/index.md

```diff
@@ -0,0 +1,220 @@
+### Introduction
+
+Node.js `Read-Evaluate-Print-Loop`(REPL) is an easy-to-use command-line tool for processing Node.js expressions.
+It captures the user's JavaScript code inputs, interprets and evaluates the result of this code, displays the result to the screen, and repeats the process till the user quits the shell.  
+It's also important to note that this tool does not require file creation to write codes. It comes with ready to use Node.js development environment.
+
+In this tutorial, we learn the basics of Node.js REPL, how we can use this amazing tool to run scripts without creating `.js` files.
+
+### Table of contents
+1. Introduction to REPL
+2. Basic Arithmetic operations using REPL
+3. Creating methods and variables using REPL
+
+### Prerequisites
+
+This tutorial requires:
+1. Node.js Installed in your development local machine.
+2. Basic knowledge of JavaScript and Node.js.
+
+### Objectives
+At the end of this tutorial, you should be able to use REPL tool to run your Node.js scripts.
```

Comment:
In the objectives section, elaborate in 3-4 lines on what we are doing in this article. For example, you can include your h4 subheadings and explain them. This ensures that the reader gets a good overview of what they're gonna be learning.